vernac/metasyntax.mli

```diff
@@ -24,6 +24,11 @@ val add_notation : locality_flag -> constr_expr ->
 
 val add_notation_extra_printing_rule : string -> string -> string -> unit
 
+(** Fine-grained activation and deactivation of notations *)
+
+val notation_activation :
```

Comment:
Thanks, finally I tweaked the low-level API to make it safer, see https://github.com/coq/coq/compare/trunk...ejgallego:Lionel-Rieg-selective-unfolding2#diff-a891e1226afa2ea34917c17845229f36R86 , now the bad case is pushed up in the code structure.